src/components/resource.js

```diff
@@ -10,8 +10,8 @@ import ResultUI from "./result-ui";
 import {LocationContext} from "../contexts/location";
 
 const Resource = () => {
-  const { responseDocument, onEntryPoint } = useContext(LocationContext);
-  const { describedBy: _, ...resourceLinks } = !onEntryPoint && responseDocument ? responseDocument.getLinks() : {};
```

Comment:
I think you still need this check for `onEntryPoint` -now I'm seeing those links in the resource component